views/layout.pug

```diff
@@ -25,10 +25,10 @@ html
 
       div(class='connectWithUs')
         h3(class="connectWithUs-h3") Connect With Us
-        div: a(href="https://github.com/guipace"): img(src="/images/githubicon.png", alt="Guilherme Pace")
-        div: a(href="https://github.com/HeroRoku"): img(src="/images/githubicon.png", alt="Jevon Timmons")
-        div: a(href="https://github.com/lanejohns"): img(src="/images/githubicon.png", alt="Lane Johns")
-        div: a(href="https://github.com/MattMav21"): img(src="/images/githubicon.png", alt="Matthew Hasan")
+        div: a(href="https://github.com/guipace"): img(class="connectWithUs-github" src="/images/githubicon.png", alt="Guilherme Pace")
```

Comment:
Please change this (and the following three github profile links) into mixins to DRY up this section.